src/test/java/hudson/plugins/parameterizedtrigger/test/BuildTriggerConfigTest.java

```diff
@@ -121,6 +132,65 @@ public void testGetProjectListStatic() throws Exception {
 
     }
 
+    public void testGetProjectListWithWorkflow() throws Exception {
+        Project<?, ?> masterProject = createFreeStyleProject("project");
+        WorkflowJob p = (WorkflowJob) Jenkins.getInstance().createProject(WorkflowJob.class, "workflowproject");
+        p.setDefinition(new CpsFlowDefinition("println('hello')"));
+
+        // Trigger a normal and workflow project
+        BlockableBuildTriggerConfig masterConfig = createConfig("subproject1, workflowproject");
+        addParameterizedTrigger(masterProject, masterConfig);
+
+        // Only create 1 sub project
+        Project subProject1 = createFreeStyleProject("subproject1");
+        subProject1.setQuietPeriod(0);
+
+        List<Job> jobs = masterConfig.getProjectListGeneral(masterProject.getParent(), null);
+        assertEquals(2, jobs.size());
+        assertTrue("Job should include workflow job", jobs.contains(p));
+        assertTrue("Job should include non-workflow job", jobs.contains(subProject1));
+
+        List<AbstractProject> projects = masterConfig.getProjectList(masterProject.getParent(), null);
+        assertEquals(1, projects.size());
+        assertFalse("Projects should NOT include workflow job", projects.contains(p));
+        assertTrue("Projects should include non-workflow job", projects.contains(subProject1));
+    }
+
+    public void testBuildWithWorkflowProjects() throws Exception {
+        Project<?, ?> masterProject = createFreeStyleProject("project");
+        WorkflowJob workflowProject = (WorkflowJob) Jenkins.getInstance().createProject(WorkflowJob.class, "workflowproject");
+        workflowProject.setDefinition(new CpsFlowDefinition("node { echo myParam; }"));
+
+        // Trigger a normal and workflow project
+        String projectToTrigger = "subproject1, workflowproject";
+        List<AbstractBuildParameters> buildParameters = new ArrayList<AbstractBuildParameters>();
+        buildParameters.add(new CurrentBuildParameters());
```

Comment:
I am wondering why you are using `CurrentBuildParameters` at all, when you are passing no parameters to `masterProject.scheduleBuild2`. The test would seem to stand on just `PredefinedBuildParameters`.